test/transactions/4_registerMultisignatureAccount.js

```diff
@@ -348,10 +347,9 @@ describe('#registerMultisignatureAccount transaction', () => {
 			});
 
 			it('should have the asset with the multisignature with the min, lifetime and keysgroup', () => {
-				return registerMultisignatureTransaction.should.have
-					.property('asset')
-					.with.property('multisignature')
-					.with.properties('min', 'lifetime', 'keysgroup');
+				return registerMultisignatureTransaction.should.have.nested
+					.property('asset.multisignature')
+					.and.to.have.all.keys('min', 'lifetime', 'keysgroup');
```

Comment:
I think `with` is clearer than `and` when we're moving to making assertions on the nested value.